src/helpers.js

```diff
@@ -135,3 +137,32 @@ export const truncate = (a, b, c, d, ellipsis) => {
 
     return i
 }
+
+/**
+ * Pager truncation Remote algorithm
+ */
+export const truncateRemote = (a, b, c, d, ellipsis) => {
+
+    const i = []
+
+    a.forEach(c => {
+        let d = c.label
+        let f;
+        if (d!=='...' && d!=='&laquo; Previous' && d!=='Next &raquo;'){
```

Comment:
This looks like it's requiring the usage of the English words "Previous" and "Next" and therefore won't work if users use terms in other languages. Is that the case?